src/task.c

```diff
@@ -10,23 +10,29 @@
 
 int              hook_init_done = 0;
 static int       num_hooks      = 0;
-static task_hook hooks[MAX_HOOKS];
+// RESOLVE: At 8 bytes per task_hook{}, we are burning up 16K of memory. Only 2 hooks
+// are currently being connected. What about reducing MAX_HOOKS to, say, 8?
+static task_hook hooks[MAX_HOOKS];	
 
 // forward declarations that aren't part of the public API of the task system
-platform_status
+static platform_status
```

Comment:
Many changes in this file add `static` to functions / prototypes to document clearly that this is a static function.
Improves code readability.